test/GithubApi.Gist.Fetch.test.js

```diff
@@ -0,0 +1,70 @@
+const { StatusCodes } = require('http-status-codes');
+const chai = require('chai');
+const { expect } = require('chai');
+const chaiSubset = require('chai-subset');
+
+require('isomorphic-fetch');
+
+chai.use(chaiSubset);
+
+const path = 'https://api.github.com';
+let response;
+let rta;
+let getGist;
+
+const promise = 'new Promise((resolve, reject) => {resolve("isomorphic-fetch");})';
+
+const gist = {
+  description: 'Example of a gist with isomorphic-fetch ',
+  public: true,
+  files: {
+    'testFile.js': {
+      content: promise
+    }
+  }
+};
+
+const config = {
+  method: 'POST',
+  body: JSON.stringify(gist),
+  headers: { Authorization: `token ${process.env.ACCESS_TOKEN}` }
+};
+
+describe('Consuming DELETE Method with isomorphic-fetch', () => {
+  before(async () => {
+    response = await fetch(`${path}/gists`, config);
+    rta = await response.json();
+  });
+
+  it('Creating and Verifying Gist', () => {
+    expect(response.status).to.equal(StatusCodes.CREATED);
+    expect(rta).to.containSubset(gist);
+  });
+
+  describe('Get the Gist', () => {
+    before(async () => {
+      getGist = await fetch(`${rta.url}`, { method: 'GET', headers: { Authorization: `token ${process.env.ACCESS_TOKEN}` } });
```

Comment:
Try to skip lines to it to be more readable, something like:
```suggestion
      getGist = await fetch(`${rta.url}`, {
        method: 'GET', 
        headers: { Authorization: `token ${process.env.ACCESS_TOKEN}` } });
```

Same for other requests